tests/cpp_methods/cppfilter_test.cc

```diff
@@ -176,6 +180,8 @@ filter_basic2::getOutputDim (GstTensorsInfo *info)
   info->info[0].dimension[1] = 16;
   info->info[0].dimension[2] = 16;
   info->info[0].dimension[3] = 2;
+  for (int i = 4; i < NNS_TENSOR_RANK_LIMIT; i++)
+    info->info[0].dimension[i] = 1;
   return 0;
```

Comment:
LGTM !! and How about below code?
```
int value[NNS_TENSOR_RANK_LIMIT] = {3, 16, 16, 2, 1, 1, 1, 1};
  for (int i = 0; i < NNS_TENSOR_RANK_LIMIT; i++)
    info->info[0].dimension[i] = value[i];
return 0;
```

Reply:
This PR is for not to use hardcoded dimension.
So I think `{3, 16, 16, 2, 1, 1, 1, 1}` this part may cause a problem when NNS_TENSOR_RANK_LIMIT has a different value.
Currently, NNS_TENSOR_RANK_LIMIT is 4 in main branch :)

Reply:
Thank you.
In that case, I think you can use value[8].